client/src/preview/route/digitaltwins/editor/PreviewTab.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { Remarkable } from 'remarkable';
+import 'katex/dist/katex.min.css';
+// @ts-expect-error: Ignoring TypeScript error due to missing type definitions for 'remarkable-katex'.
+import RemarkableKatex from 'remarkable-katex';
+import SyntaxHighlighter from 'react-syntax-highlighter';
+
+interface PreviewProps {
+  fileContent: string;
+  fileType: string;
+}
+
+function PreviewTab({ fileContent, fileType }: PreviewProps) {
```

Comment:
1. The md, json and yaml cases are handled. What happens if it is lifecycle/script?
2. The hidden files (starting with dot character) should also be available. This allows us to edit `.gitlab-ci.yaml` file as well.

Reply:
1. Added case for handling bash files, without extension.

Reply:
It is not just the bash files. Please show all the files placed in `lifecycle` directory